pkg/operator/csi/csidrivercontroller/csi_driver_controller_test.go

```diff
@@ -124,6 +124,7 @@ func newOperator(test testCase, t *testing.T) *testContext {
 		controllerName,
 		operandName,
 		operandNamespace,
+		"cluster",
```

Comment:
Do we want to change the name away from `cluster` in this e2e test? Will this cause issues if other tests are running in parallel? For instance, if we ever get the EFS CSI driver and both are used in an AWS cluster?

Reply:
This is unit test, the CR name IMO does not really matter there.